crates/oxc_linter/src/snapshots/default_case_last.snap

```diff
@@ -0,0 +1,104 @@
+---
+source: crates/oxc_linter/src/tester.rs
+assertion_line: 105
+expression: default_case_last
+---
+  ⚠ eslint(default-case-last): Enforce default clauses in switch statements to be last
+   ╭─[default_case_last.tsx:1:1]
+ 1 │ switch (foo) { default: bar(); break; case 1: baz(); break; }
```

Comment:
nit: span the `default` keyword only, the body is uninteresting regarding the error. You can use `Span::new(default_clause.span.start, default_clause.span.start + 7)`.

Reply:
Get.